### DISCUSSION TITLE
Setting different options in the `/etc/krb5.conf` placed by ipaclient?

### DISCUSSION THREAD

**Discussion by Drugantibus** [freeipa/ansible-freeipa#1110] [General]
Is it possible to modify the values for certain options contained in `/etc/krb5.conf` when this file is created by the `ipaclient` role? Specifically, I'd like to set `rdns` and `dns_canonicalize_hostname` to `true` to avoid "host not found in database" errors when connecting using shortnames or IP addresses. Or is there a different way to solve this issue? 
Apologies if this is documented or already answered - couldn't find anything. Thanks!

**Comment by abbra** [comment]
`/etc/krb5.conf.d` is included by default on most platforms. Just drop a file to `/etc/krb5.conf.d/` with the required configuration snippet. Below is an example what FreeIPA itself does to enable SPAKE:
```
$ cat /etc/krb5.conf.d/freeipa 
[libdefaults]
    spake_preauth_groups = edwards25519

```
